src/components/link/AbstractLink.component.jsx

```diff
@@ -45,41 +45,41 @@ const AbstractLink = React.createClass({
 			nextProps.target !== this.props.target ||
 			nextProps.targetHandlePosition !== this.props.targetHandlePosition;
 	},
-	renderLinkSourcehandle(){
-		if(this.props.linkSourceHandleComponent){
-			return <LinkHandle
+	renderLinkSourcehandle() {
```

Comment:
I think this could be a pure component outside of this one which just group + render LinkSourceHandler and LinkTargetHandler

By the way did you do a typo here ? handle**R** ?